static/styles/billing.scss

```diff
@@ -39,7 +39,7 @@
     }
 
     .tab-content {
-        border: 1px solid #ddd;
+        border: 1px solid hsl(0, 0%, 87%);
```

Comment:
In the past seems like we converted this to hsl(0, 0%, 86%);
https://github.com/zulip/zulip/commit/a5328be9a1fd3#diff-bf5df85ce1789e5122c7310986fd0d6eR133